app/helpers/date_range_helper.rb

```diff
@@ -22,7 +22,7 @@ def selected_interval
   end
 
   def selected_range
-    now = Time.zone.now
+    now = Time.zone.now.end_of_day
```

Comment:
I _think_ this is an improvement, but I'm open to skip this bit.

Reply:
I'm not opposed to the wording, but what does "This week to date" mean? Is that like "Today is wednesday, and this includes Sun -> Today from this week"?